.travis/install_ubuntu_dependencies.sh

```diff
@@ -20,19 +20,19 @@ sudo apt-get update
 
 DEPENDENCIES="indent kwstyle"
 
-sudo apt-get install -y ${DEPENDENCIES}
+sudo apt-get install -y "${DEPENDENCIES}"
```

Comment:
Again, this is overly eager quoting. There is no package called "indent kwstyle", but instead these are two packages.